mysql-test/suite/galera/t/pxc_strict_mode.test

```diff
@@ -0,0 +1,1189 @@
+
+#
+# Test different options with different variation of pxc-strict-mode.
+#
+
+--source include/galera_cluster.inc
+--source include/have_innodb.inc
+
+
+#-------------------------------------------------------------------------------
+#
+# create test-bed
+#
+# let's cache the default value first.
+--connection node_1
+--echo #node-1
+--let $pxc_strict_mode_saved1 = `SELECT @@pxc_strict_mode`
+--let restartdir= $MYSQLTEST_VARDIR/tmp/restartdir
+--mkdir $restartdir
+
+--connection node_1
```

Comment:
At the bottom of the test it restores pxc_strict_mode_saved2 for connection node_2, however it's saving the value of connection node_1 into pxc_strict_mode_saved2.

so, from line 21, shouldn't this be
--connection node_2
--echo #node-2
--let $pxc_strict_mode_saved2 = `SELECT @@pxc_strict_mode`